lib/msgServer/msgStream/transports/http-polling/index.js

```diff
@@ -105,7 +105,7 @@ HttpPollingHost.prototype.deliver = function (msgs) {
 
 
 HttpPollingHost.prototype.respondBadRequest = function (reason) {
-	if (!this.res) {
+	if (this.res.finished) {
```

Comment:
I believe the code I replaced it with has the same behavior, except in the cse where we would send before the res is set - which would be something we would definitely want to see pop up as an error I think.